index.js

```diff
@@ -187,13 +187,12 @@ module.exports = function (opts) {
       function setupRPC (stream, manf, isClient) {
         var rpc = Muxrpc(create.manifest, manf || create.manifest)(api, stream.auth)
         var rpcStream = rpc.createStream()
-        if(timeout_inactivity > 0) rpcStream = Inactive(rpcStream, timeout_inactivity)
+        var id = rpc.id = '@'+u.toId(stream.remote)
+        if(timeout_inactivity > 0 && id !== rpc.id) rpcStream = Inactive(rpcStream, timeout_inactivity)
```

Comment:
this isn't right, `id !== rep.id` can't ever be true because `id = rpc.id` is set on line 190